Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -1447,9 +1447,18 @@ std::vector<FCesiumCamera> ACesium3DTileset::GetEditorCameras() const {
         !pEditorViewportClient->IsPerspective()) {
       continue;
     }
-
+    auto GetViewRotation = [pEditorViewportClient]() {
```

Comment:
Is this lambda needed? At first I thought it was saving a copy of the FRotator, since all we really need is a const reference. But it looks like the inferred return value of the lambda function is just a plain FRotator (one of the code paths returns a const ref but the other returns a FRotator by value). I actually didn't know a temporary value (the result of the lambda call) could be assigned to a const ref (e.g., on L1461)! But I still think a copy is happening regardless. If that is the case, would it be better to turn that const ref into a temporary variable and use a straightforward conditional to choose an FRotator, instead of the lambda function?

Or is the compiler optimizing something that I don't understand?

Reply:
Thanks, that does make sense.